src/components/Petitions/index.js

```diff
@@ -0,0 +1,19 @@
+import React from 'react';
+import PetitionListItem from 'components/PetitionListItem';
+import Pager from 'components/Pager';
+
+const Petitions = ({ total, petitions, page, per }) => (
+  <div>
+    <ul>
+      {petitions.map((petition, index) => {
+        return (
+          <PetitionListItem {...petition} key={petition.id} />
+        );
+      })}
+    </ul>
+
+    <Pager {...{ path: '/petitions', page: page, per: per, total: total }} key={page} />
```

Comment:
prefer

```
<Pager
  path: '/petitions'
  page: page
  per: per
  total: total
/>
```

don't think we need `key`? that's just if you are iterating over components :)